packages/psss-server/src/models/PsssSession.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable max-classes-per-file */
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { DatabaseModel, DatabaseType } from '@tupaia/database';
+import { AccessPolicy } from '@tupaia/access-policy';
+
+export class PsssSessionType extends DatabaseType {
+  static databaseType = 'psss_session';
```

Comment:
Spell `PsssSession` fast 7 times in a row 😝

Reply:
Haha I thought the same! Originally I had it with capital S's, but that was just impossible to parse